Tasks/NotationV0/task.json

```diff
@@ -0,0 +1,248 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Microsoft/azure-pipelines-task-lib/master/tasks.schema.json",
+  "id": "a7d56ba6-35a6-453d-8192-8b602bf234bd",
+  "name": "Notation",
+  "friendlyName": "Notation",
+  "description": "Azure Pipepine Task for setting up Notation CLI, sign and verify with Notation",
+  "helpMarkDown": "",
+  "category": "Utility",
+  "author": "Notation Developers",
```

Comment:
please add "minimumAgentVersion": "2.144.0", see [ref](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/migrateNode16.md#upgrading-tasks-to-node-16)

Reply:
Updated.

Reply:
Many thanks @DmitriiBobreshev !